### DISCUSSION TITLE
Andoid "home and overview buttons" effect after click on ads

### DISCUSSION THREAD

**Discussion by orah29** [Poing-Studios/Godot-AdMob-Android-iOS#186] [Q&A]
With the sample project supplied with the plugin, after clicking on ads (banner or interstitial) :
if I press on home/overview button, the app closes;
if I press on back button, I return on ads. Then I close the ads to return to the app.
How to fix this?
Thanks in avance

**Comment by gumaciel** [comment]
See https://github.com/Poing-Studios/godot-admob-android/issues/187
